replay_log.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+import os
+import requests
+
+
+CLIENT_UPLOAD_FOLDER = "client_upload/"
+CLIENT_DOWNLOAD_FOLDER = "client_download/"
+SERVER_URL = "http://localhost:5000"
+
+def write_file(filename):
+  write_url = SERVER_URL + "/write"
+  if not os.path.exists(filename):
+    return None
+  files = {'file': open(filename, 'rb')}
+  # may need get latency number
+  r = requests.put(write_url, files = files)
+  if (r.status_code != requests.codes.created):
+    # request failed
+    return None
+  file_uuid = r.text
+  return file_uuid
+
+def read_file(file_uuid):
+  read_url = SERVER_URL + "/read?uuid=" + file_uuid
```

Comment:
You can do something like this `url = 'http://%s/read?%s' % (url, urllib.urlencode({ 'uuid': filename }))`